Packages/com.unity.inputsystem/Documentation~/Installation.md

```diff
@@ -19,13 +19,13 @@ To install the new Input System, open Unity's package manager (menu: __Window >
 
 ## Enabling the new input backends
 
-By default, Unity's classic Input Manager (`UnityEngine.Input`) is active and support for the new Input System is inactive. This allows existing Unity Projects to keep working as they are.
+By default, Unity's classic Input Manager (`UnityEngine.Input`) is active, and support for the new Input System is inactive. This allows existing Unity Projects to keep working as they are.
 
-When you install the Input System package, Unity will ask whether you want to enable the new backends. If you click **Yes**, Unity will enable the new backends and disable the old backends, and the Editor will restart.
+When you install the Input System package, Unity will ask whether you want to enable the new backends. Click **Yes** to enable the new backends and disable the old backends. The Editor restarts during this process.
```

Comment:
Is this still really correct? Isn't the default behavior that BOTH backends are active and the restart is not needed? We hope it is and if that is the case we might have to refer to this behavior for Unity version where this is still true and mention from which version default changed to BOTH.,